src/Domain.Linnapps/Products/ProductRange.cs

```diff
@@ -0,0 +1,32 @@
+﻿namespace Linn.Products.Domain.Linnapps.Products
+{
+    using System;
+    using System.ComponentModel.DataAnnotations.Schema;
+
+    using Linn.Common.Domain.Exceptions;
+
+    public class ProductRange
+    {
+        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
```

Comment:
What is this doing?

Reply:
Was trying to get it to allow Oracle to set the id in the database. I've removed it now.
It's an unsolved problem.

Reply:
I was going to bring it up in show and tell